<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>881</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>AUTO-CARE</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;Matricule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="matriculeInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;marque&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;Kilométrage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;Modéle&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;Type de la panne&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;Date d'arrivage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>150</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;Date de sortie&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="kilometrageInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="modeleInput">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>80</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="marqueInput">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateArrivageInput">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateSortieInput">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>150</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="typePanneInput">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mécanique&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Mécanique</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="vidangeInput">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>210</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mécanique&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>vidange</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="autreInput">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>210</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mécanique&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Autre</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="electriqueInput">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mécanique&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>électrique</string>
     </property>
    </widget>
    <widget class="QComboBox" name="idProprietaireInput">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>260</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>l'ouvrier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ahmed miled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>yassin benhamed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>aymen sakouhi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>youssef akrout</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>851</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/Design sans titre (3) (1).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>540</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; font-style:italic; text-decoration: underline; color:#5555ff;&quot;&gt;Kilométrage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouterButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimerButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>370</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifierButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>440</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <zorder>label_8</zorder>
    <zorder>label</zorder>
    <zorder>matriculeInput</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>kilometrageInput</zorder>
    <zorder>modeleInput</zorder>
    <zorder>marqueInput</zorder>
    <zorder>dateArrivageInput</zorder>
    <zorder>dateSortieInput</zorder>
    <zorder>typePanneInput</zorder>
    <zorder>vidangeInput</zorder>
    <zorder>autreInput</zorder>
    <zorder>electriqueInput</zorder>
    <zorder>idProprietaireInput</zorder>
    <zorder>label_9</zorder>
    <zorder>ajouterButton</zorder>
    <zorder>supprimerButton</zorder>
    <zorder>modifierButton</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>marqueInput</tabstop>
  <tabstop>vidangeInput</tabstop>
  <tabstop>idProprietaireInput</tabstop>
  <tabstop>dateArrivageInput</tabstop>
  <tabstop>dateSortieInput</tabstop>
  <tabstop>matriculeInput</tabstop>
  <tabstop>typePanneInput</tabstop>
  <tabstop>kilometrageInput</tabstop>
  <tabstop>autreInput</tabstop>
  <tabstop>electriqueInput</tabstop>
  <tabstop>modeleInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
